pages.zh/android/logcat.md

```diff
@@ -0,0 +1,16 @@
+# logcat
+
+> 转储系统消息日志。
+> 更多信息见：https://developer.android.com/studio/command-line/logcat>.
+
+- 显示系统日志：
+
+`logcat`
+
+- 将系统日志写入文件：
+
+`logcat -f {{path/to/file}}`
```

Comment:
You can translate the argument option to help non-English speakers.

Reply:
like this: https://github.com/tldr-pages/tldr/pull/6019#discussion_r637554415

We can discuss it in the remarks above. After we have a conclusion, I can revise these files in a unified way. @navarroaxel

Reply:
`file` has been translated @navarroaxel

Reply:
I don't get it, in this case I think in someting similar too:
```suggestion
`logcat -f {{文件路径}}`
```

Reply:
Oh, I see, you mean that the placeholders in the example commands must be translated into Chinese, right?

In this example, there is no problem, but it may be more difficult for people to understand the placeholders translated into Chinese in other scenarios.